src/app/app.component.html

```diff
@@ -3,4 +3,51 @@ <h1>
     Najlepsza {{ title }} w sieci!
   </h1>
   <img width="300" alt="Angular Logo" src="http://vitaltrading.pl/wp-content/uploads/2016/04/checklist-1000x675.jpg">
-</div>
\ No newline at end of file
+</div>
+
+
+<input [(ngModel)]="newTask" placeholder="Wpisz zadanie do wykonania" (keyup.enter)='pushTask()' class="addTask">
+<button (click)="pushTask()">Dodaj Zadanie</button>
+<button type='sort' (click)=sortAZ()>Alfabetycznie</button>
```

Comment:
W HTML używaj `"` zamiast `'`. Jeśli podstawiasz cokolwiek od czegokolwiek tak jak w tym przypadku bindujesz do outputu `click` to również otaczaj to w `"`. Ta linijka w poprawny sposób powinna wyglądać tak:
```html
<button type="sort" (click)="sortAZ()">Alfabetycznie</button>
```

Zastosuj to wszędzie.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/66add8f495af27f61436b5c81d1539a320b0fd83